source/_components/binary_sensor.doorbird.markdown

```diff
@@ -0,0 +1,30 @@
+---
+layout: page
+title: "DoorBird Binary Sensor"
+description: "Instructions how to integrate DoorBird video doorbell state into Home Assistant."
+date: 2017-08-06 11:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: doorbird.png
+ha_category: Binary Sensor
+ha_release: "0.50"
+ha_iot_class: "Local Polling"
+---
+
+The `doorbird` binary sensor platform allows Home Assistant to monitor when your [DoorBird](http://www.doorbird.com/) doorbell rings.
+
+<p class='note'>
+  You must have the [DoorBird component](/components/doorbird/) configured to use this binary sensor.
+</p>
+
+To enable the binary sensor, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+binary_sensor:
+  - platform: doorbird
+    monitored_conditions:
+      - doorbell
+```
```

Comment:
Definition of configuration variables are missing. Suggest you consult other [similar components](https://home-assistant.io/components/binary_sensor.mqtt/) for examples.